docker/docker-compose.yml

```diff
@@ -23,5 +23,23 @@ services:
       - '3000:3000'
     environment:
       - SNACK_SHOP_BE_URL=http://snack-shop-be:8080/
+    depends_on:
+      - snack-shop-be
+
+  snack-shop-fe:
+    build: ../snack-shop-fe
+    restart: always
+    ports: 
+      - '9090:9090'
+    environment:
+      - SNACK_SHOP_BFF_URL=http://snack-shop-bff:3000/ # TODO: Use this in the FE
+    depends_on:
+      - snack-shop-bff
+  
+  stock-snacks:
+    build: ../dev
+    restart: no
+    environment:
+      - SNACK_API_ENDPOINT=http://snack-shop-be:8080/api/snacks
```

Comment:
Starting this service for the first time, it fails to connect to the backend, even though it waits for the container running the backend service is not yet. Starting the snack stocker a second time it establishes the connection. Let's add the backend's `/actuator/health` endpoint as docker compose [healthcheck](https://docs.docker.com/compose/compose-file/05-services/#healthcheck) for the backend service being considered ready to accept traffic.